src/notifier/gitlab.rs

```diff
@@ -4,14 +4,15 @@ use crate::template::Template;
 use anyhow::{Context, Result};
 use gitlab::api::projects::merge_requests::notes::{EditMergeRequestNote, MergeRequestNotes};
 use gitlab::api::{self, projects::merge_requests::notes::CreateMergeRequestNote, Query};
-use gitlab::types::Note;
```

Comment:
types module was deleted in https://gitlab.kitware.com/utils/rust-gitlab/-/merge_requests/453

And, in docs, we should declare own struct in order to fetch what we need. https://docs.rs/gitlab/0.1700.1/gitlab/api/index.html